extensions/common/xwalk_external_extension.h

```diff
@@ -39,8 +40,9 @@ class XWalkExternalExtension : public XWalkExtension {
 
   bool Initialize();
 
-  void set_runtime_variables(const base::ValueMap& runtime_variables) {
-    runtime_variables_ = runtime_variables;
+  void set_runtime_variables(const base::DictionaryValue::Storage& runtime_variables) {
+    runtime_variables_.emplace_hint(runtime_variables_.end(), runtime_variables.begin()->first,
```

Comment:
Don't understand why you need to use emplace_hint here. Don't find any comparable use in the upstream patch you referenced in the commit message.